engine/src/main/java/org/destinationsol/ResolutionProvider.java

```diff
@@ -17,9 +17,11 @@
 
 import com.badlogic.gdx.Graphics;
 
-import java.util.Comparator;
-import java.util.List;
+import java.util.*;
```

Comment:
Hi, please do not use star imports (It's part of our coding convention, we should definitely write this somewhere here @vampcat)

Reply:
Done